packages/cozy-harvest-lib/src/helpers/konnectors.js

```diff
@@ -172,7 +172,7 @@ export const getErrorLocale = (
 
 export const fetchSupportMail = async client => {
   const result = await client.fetchQueryAndGetFromState({
-    definition: Q('io.cozy.settings').getById('context'),
+    definition: Q('io.cozy.settings').getById('io.cozy.settings.context'),
     options: {
       as: 'contextSupportMail',
```

Comment:
This is not the right `as`. Should be `io.cozy.settings/io.cozy.settings.context` (without the right as, cache is not working correctly since the dedup is done on the as)

Reply:
Right I did not even read it. Let's fix it in the same time.